### ISSUE TITLE
Preview Message Links Not Working

### ISSUE THREAD

**Issue by zluckytraveler** [dylan-dang/BetterDiscordPlugins#16]
The new Discord update broke the Preview Message Links plugin unfortunately.

**Comment by dylan-dang** [comment]
yeah I haven't gotten around to fixing it as I'm busy with college, and change to swc mangled exports has made it harder to find the necessary discord modules. I'll get around it to it, but pull requests would be greatly appreciated.

**Comment by yofukashino** [comment]
- I will try to get it working then

**Comment by zluckytraveler** [comment]
@Tharki-God @dylan-dang Did either of you guys end up getting a chance to rectify this plugin to start working again? It's by far one of my most favorite used plugins, and the last one I know of that was working that did this.

**Comment by KaKi87** [comment]
Hello,
Any news on this ?
Thanks
